roles/prerequisites/tasks/main.yml

```diff
@@ -1,4 +1,28 @@
 ---
 - include_tasks: required-packages.yml
-- include_tasks: python-stuffs.yml
+  tags:
+      - shell
+      - networking
+      - devops
+      - containers
+      - utilities
+      - development
+      - media
+      - communication
+      - monitoring
```

Comment:
The abundance of tags complicates the development, memorization, and maintenance of the playbook. I am a strong proponent of a minimalist approach, which I believe would streamline these processes.

As an example, why do we use containers! I believe it is dedicated to the DevOps tag. Also the other tags sounds a little bit confusing for me, i.e. networking, utilities, development, etc.

Reply:
My preferred tags are:

```
mostafa-preferred-tags:
  - devops
  - shell
  - developer
  - communication
```

Reply:
Since we have vlc and also telegram package and they are not in the same category (communication), I've selected the same tag for them as "desktop" if you agree, let me know please, @mostafaghadimi

Reply:
I do agree.